src/main/java/com/esri/samples/portal/integrated_windows_authentication/README.md

```diff
@@ -0,0 +1,42 @@
+# Integrated windows authentication
+
+Use Windows credentials to access services hosted on a portal secured with Integrated Windows Authentication (IWA).
+
+![](IntegratedWindowsAuthentication.png)
+
+## Use case
+
+Integrated Windows Authentication, which is built into Microsoft Internet Information Server (IIS), allows users to easily authenticate for different services using their domain's Windows credentials. This simplifies logging in to services that are provided through an intranet environment, as only a single username and password combination are required for the applications that support Integrated Windows Authentication.
```

Comment:
this is much better, nice! But I think we still need something more specific. Consider a company that needs authentication for its staff to be able to access and manage data hosted on ArcGISOnline: is that a good place to start for IWA? If so, try and follow the use case through there. 

(side note, since the meaning of the IWA acronym is described fully in the first sentence of the README, then to shorten the words in the Use case, you can refer to it as IWA at the beginning of the sentence (and the same for its usage at the end of the section)).